update_llm.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# update_llm.sh
```

Comment:
Perhaps `update_ollama_models.sh` would be more appropriate and descriptive for the name.

Reply:
Great suggestion, file name changed